core/market/src/matcher.rs

```diff
@@ -86,12 +86,11 @@ impl Matcher {
         // until first change to value will be made.
         counter!("market.offers.incoming", 0);
         counter!("market.offers.broadcasts", 0);
-        counter!("market.offers.broadcasts.size", 0);
+        counter!("market.offers.broadcasts.skip", 0);
```

Comment:
removing `size` metrics here and adding a new one, to measure `skip`ed ones, which is crucial for core team to decide how to deal with bcasts in the future.